client/views/7-ShowContactProfileView.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, Text, Linking } from 'react-native';
+import { Avatar, Icon } from 'react-native-elements';
+
+import HeadersActions from './../components/HeadersActions';
+
+const canOpenURL = (url) => {
+  Linking.canOpenURL(url).then(supported => {
+    if (!supported) console.log('Can\'t handle url: ' + url);
+    else return Linking.openURL(url);
+  }).catch(err => console.error('An error occurred', err));
+};
+
+const shareProfileToQRC = (contact) => {
+  console.log('+++', JSON.stringify(contact));
+}
+
+export default class ContactProfileView extends Component {
+  static navigationOptions = {
+    title: 'Add Contact',
+    headerRight: (
+      <HeadersActions></HeadersActions>),
+  };
+
+  render() {
+    const { navigation } = this.props;
+    let contact = navigation.getParam('contact');
+
+    const { fName, lName, occupation, currentLocation: loc, email, birthplace  } = contact.personal;
+    const { facebook, instagram, twitter, blog } = contact.social;
+    const { linkedIn, github, cv, website } = contact.networking;
+    const {photo} = contact;
+    return (
+      
+      <View style={styles.container}>
+        <View style={styles.contactsWrapper}>
+          <Avatar
+            xlarge
+            rounded
+            source={{ uri: photo }}
+            onPress={() => console.log("Works!")}
+          />
+          <Text style={styles.name}> {`${fName} ${lName}`} </Text>
+          <Text style={styles.jobtitle}> {`${occupation}`} </Text>
+          <Text style={styles.details}>
+            Lives in - {`${loc.country}, ${loc.place}`}
+          </Text>
+          {/* <Text style={styles.details}>from {`${birthplace.country}`} </Text> */}
+
+          <Text 
+            style={styles.website}>
+            {`${website}`}
+          </Text>
+
+        </View>
+        <Text style={styles.category}> Social </Text>
+        <View style={styles.iconsContainer}>
+          <Icon
```

Comment:
create a seperate Icon component and you would reduce the repetitiveness by 90% haha